src/ui/views/ViewServers.qml

```diff
@@ -88,9 +89,63 @@ Item {
                     }
                 }
 
+                VPNSearchBar {
+                    id: serverSearchInput
+
+                    anchors.left: parent.left
+                    anchors.right: parent.right
+                    anchors.leftMargin: Theme.vSpacing
+                    anchors.rightMargin: Theme.vSpacing
+                    enabled: true
+                    height: Theme.rowHeight
+                    onTextChanged: () => {
+                        countriesModel.invalidate();
+                    }
+                    placeholderText: VPNl18n.tr(VPNl18n.ServersViewSearchPlaceholder)
+                    stateError: countriesRepeater.count === 0
```

Comment:
In case #1459 gets merged before this PR this has to be changed to `hasError: countriesRepeater.count === 0` @lesleyjanenorton.